index.html

```diff
@@ -19,5 +19,115 @@ <h1 class="title">Michael<br/>Olusola</h1>
       If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
     <button class="button" type="submit">Start collaboration</button>
   </section>
+  <section class="projects">
+    <h2 class="project-heading">Projects</h2>
+    <article class="project-card">
+      <img class="project-image" src="./images/Rectangle-21.png" alt="project image"/>
+      <div class="project-details">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="language-list">
+          <li>HTML/CSS</li>
+          <li>Ruby on Rails</li>
+          <li>JavaScript</li>
+        </ul>
+        <button class="project-button" type="submit">See this project &rarr;</button>
```

Comment:
- [ ] You have done an amazing job styling your buttons. There is one small thing left right now. Kindly add the button interaction states for your buttons eg on click, on hover, etc. Currently, your buttons feel very static and are not intuitive. If you may be having doubts of how to achieve this, have a look at [this wonderful resource](https://zellwk.com/blog/style-hover-focus-active-states/).

**The button states from your chosen Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185754663-0237d057-897c-4db8-9593-ce013e23af21.png)


**Kindly implement this on all the buttons of the project.**

- [ ] Also, kindly note that this type of button might not be a form button. Thus having the type "submit" is not the correct type to add here. I would recommend you change it to a type "button" instead.

Reply:
Thank you for the review